core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -2967,22 +2968,28 @@ void ExHbaseAccessTcb::handleException(NAHeap *heap,
                                     ComCondition *errorCond,
                                     ExpHbaseInterface * ehi,
                                     NABoolean & LoggingFileCreated,
-                                    char *loggingFileName)
+                                    char *loggingFileName,
+                                    ComDiagsArea **loggingErrorDiags)
 {
   Lng32 errorMsgLen = 0;
   charBuf *cBuf = NULL;
   char *errorMsg;
   Lng32 retcode;
 
+  if (*loggingErrorDiags != NULL)
```

Comment:
Not quite sure I understand this logic. If a ComDiagsArea object has already been allocated, does that mean we have already reported the exception somewhere else? Or perhaps this is not the first exception and we simply don't want to log more than one?